### ISSUE TITLE
ensure all amheida videos are accessible before publication

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#410]
_governing epic: #408 migrate amheida content from standalone website to the research section of isaw web_

There are videos in the Amheida website content. General experience suggests that it's likely that none of these videos have been prepared properly for accessibility (i.e., with transcripts or closed captioning). The latter condition is a WCAG violation and must be corrected before the Amheida materials can be published.

Steps:

 - [ ] determine which videos need remediation (make a list)
 - [ ] ask Amheida team to prepare and provide transcripts or add closed captioning
 - [ ] if Amheida team provides transcripts, upload these through the plone and add appropriate tagging to alert screen-reader users to the relationship

**Comment by paregorios** [comment]
Assigned this to myself to do the first step: make a list of all amheida-related images and whether they need alt text or not.
